### DISCUSSION TITLE
Configuring proxy certificate

### DISCUSSION THREAD

**Discussion by JimmySimard** [crc-org/crc#2744] [Q&A]
I try to configure a pod with a Dockerfile but when arrive the moment to pull the first image from the docker repo, the build stop with x509 error. I have configure a ConfigMap name user-ca-bundle with my certificate but nothing change. I got the same error... Can you help me with that ?

**Comment by praveenkumar** [comment]
@JimmySimard We have some issue with proxy on current version of CRC and should be fixed in upcoming release. I would suggest you to wait till Wednesday and then try with new release.
